pyslackersweb/website/tasks.py

```diff
@@ -44,16 +44,16 @@ class Channel:
 
 async def sync_github_repositories(
     session: ClientSession, redis: RedisConnection, *, cache_key: str = GITHUB_REPO_CACHE_KEY
-) -> None:
+) -> List[Repository]:
```

Comment:
I don't think we should change the return value just for testing, I'd rather break out the function for getting the repos so we can mock that for testing what this function does. (same with the other functions)